annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/exception/GlobalExceptionHandler.java

```diff
@@ -38,18 +36,21 @@ protected I18nService getI18nService() {
   // response.setError(errorMessage);
   // return response;
   // }
+  
+  @ExceptionHandler
+  public ResponseEntity<EuropeanaApiErrorResponse> handleRequestBodyNotParsableError(
```

Comment:
please renamed back method to handleBatchUploadException, so that the method name is inline with the purpose